stream.go

```diff
@@ -120,3 +130,15 @@ func (s *Stream) TimeBase() Rational {
 func (s *Stream) SetTimeBase(r Rational) {
 	s.c.time_base = r.c
 }
+
+type PTSWrapBits int
```

Comment:
Can you remove this type? No need for it, you can use the `int` type directly

Reply:
Done.